Dockerfile.dapper

```diff
@@ -100,16 +96,16 @@ RUN cd /usr/src && \
     git clone https://github.com/rancher/liblonghorn.git && \
     cd liblonghorn && \
     git checkout 09956ed9329d51f41a85d6e06113948c947976a8 && \
-    make deb && \
-    dpkg -i ./pkg/liblonghorn_*.deb
+    make && \
+    make install
```

Comment:
Should we had a .spec file to liblonghorn and build an RPM since we use the debian package on Ubuntu.

Reply:
I think it should be fine for now, because we actually don't deliver the artifact but rather just build then install directly.